### DISCUSSION TITLE
Introduction to JavaScript ,History, JS Engines, JS Programming Paradigms | Learn.everydaykarmaa

### DISCUSSION THREAD

**Discussion by giscus** [isarojdahal/learn.everydaykarmaa.com#22] [General]
# Introduction to JavaScript ,History, JS Engines, JS Programming Paradigms | Learn.everydaykarmaa

Source Code

https://learn.everydaykarmaa.com/notes/javascript/introduction-to-javascript-history-js-engines-js-programming-paradigms

**Comment by isarojdahal** [comment]
Great
